### ISSUE TITLE
Image resizing not work correctly

### ISSUE THREAD

**Issue by Hongbo-Miao** [Canner/slate-editor-icons#28]
When you drag the right bottom handler, it drag the image too.

Tested on Chrome, Firefox and Safari.

![image](https://user-images.githubusercontent.com/3375461/37860520-4a44dee0-2ee4-11e8-8129-f8020aaeae90.png)

**Comment by chilijung** [comment]
@Hongbo-Miao yes, this is a known issue, the problem is that slate wrapped a `draggble` attribute in every node.

code source: https://github.com/ianstormtaylor/slate/blob/3339d088e14ad0bf04a674c14834179403ae369f/packages/slate-react/src/components/void.js#L79

I'm going to open a new issue in Slate to ask why. Thanks!

**Comment by chilijung** [comment]
https://github.com/ianstormtaylor/slate/issues/1726

**Comment by pizn** [comment]
@chilijung you don't need to use `draggble` attribute to handle resize image. 

You can use `mouseDown` to save the event.clientX and event.clientY, then fire to listen the `mouseMove` event and calc the x/y size to resize the image. At last listen the `mouseUp` event to save the change.

**Comment by pizn** [comment]
ok. I found you use `react-resizable`, it base `react-draggable`, so...forget what i said above. I handle image resize by handwriting `mouseDown`/`mouseMove`/`mouseUp`.
